package.json

```diff
@@ -34,10 +34,13 @@
     "install-tools": "npm ci",
     "uninstall-tools": "rm -r node_modules;",
     "pa11y-starter": "pa11y-ci --config tests/.pa11yci.js",
+    "pa11y-starter-ide": "pa11y-ci --config tests/.pa11yci.js http://127.0.0.1:8000/",
+    "backstop-starter-ide": "backstop test --config=tests/backstop/backstop-settings.js --testhost=http://127.0.0.1:8000/ --pathfile=paths --labelfile=labels",
```

Comment:
@panshulK Is there any need to pass `testhost` argument from here as we've already added condition check in `backstop-settings.js` to default use `http://127.0.0.1:8000` baseUrl.